cli/command/service/update.go

```diff
@@ -367,6 +368,10 @@ func updateService(ctx context.Context, apiClient client.NetworkAPIClient, flags
 		updateInt64Value(flagReserveMemory, &task.Resources.Reservations.MemoryBytes)
 	}
 
+	if anyChanged(flags, flagOomScoreAdj) {
```

Comment:
nit as it's a single flag we check for, this could be `flags.Chanced(flagOomScoreAdj)` (not a blocker though)